jobs/kola-openstack.Jenkinsfile

```diff
@@ -46,81 +46,90 @@ if (s3_stream_dir == "") {
     s3_stream_dir = "fcos-builds/prod/streams/${params.STREAM}"
 }
 
-cosaPod(image: params.COREOS_ASSEMBLER_IMAGE,
-        memory: "256Mi", kvm: false,
-        secrets: ["aws-fcos-builds-bot-config", "openstack-kola-tests-config"]) {
+try { timeout(time: 120, unit: 'MINUTES') {
```

Comment:
Maybe 1h for OpenStack? Looking at past runs, it seems to take ~30 mins.

Reply:
done